tests/test_config_validation/test_retagging.py

```diff
@@ -288,6 +288,27 @@ def test_reusing_multi_platform_images():
                 push:
                     repository: user1/buildrunner-test-multi-platform2
                     tags: [ 'latest' ]
+    """,
+        """
+        steps:
+            build-container:
+                build:
+                    dockerfile: |
+                        FROM {{ DOCKER_REGISTRY }}/nginx:latest
+                        RUN printf '{{ BUILDRUNNER_BUILD_NUMBER }}' > /usr/share/nginx/html/index.html
+                push:
+                    repository: docker-xeng-release.dr.corp.adobe.com/xeng/gauntlet/tracer-bullet
+                    tags: []
```

Comment:
late to the party, this is probably as good but the cases we saw this fail in omitted the tag element entirely